bin/_mocha

```diff
@@ -64,6 +64,7 @@ program
   .option('-O, --reporter-options <k=v,k2=v2,...>', 'reporter-specific options')
   .option('-R, --reporter <name>', 'specify the reporter to use', 'spec')
   .option('-S, --sort', "sort test files")
+  .option('-a, --randomize <mode>[:seed]', "randomize tests")
```

Comment:
Let's just lose the short flag.

I imagine most people will just want to randomize tests.  Are the modes necessary?

What does randomizing suites buy you?  You can only randomize top-level ones, yes?